ADAL/src/public/ADTelemetry.h

```diff
@@ -76,15 +57,15 @@
 /*!
```

Comment:
(from the comments above)

> Telemetry events will be flushed when function flush is called by developer.

Does the app developer need to call flush? Not clear how you do that from this interface.